third_party/libunwindstack/tests/MapInfoGetObjectTest.cpp

```diff
@@ -527,4 +529,306 @@ TEST_F(MapInfoGetObjectTest, read_only_followed_by_empty_then_read_exec_share_el
   ASSERT_EQ(object, r_info->GetObject(process_memory_, ARCH_ARM));
 }
 
+template <class>
+inline constexpr bool kAlwaysFalseV = false;
+
+template <typename PeCoffInterfaceType>
+class MapInfoGetObjectPeCoffTest : public ::testing::Test {
+ public:
+  MapInfoGetObjectPeCoffTest() {
+    std::string file_name;
+    if constexpr (std::is_same_v<PeCoffInterfaceType, PeCoffInterface32>) {
+      // About libtest32.dll.
```

Comment:
Do we need the comment with the numbers here? Some of them seem to be used in the test (like SizeOfHeaders), but some are not. Rebuilding libtest32.dll and libtest.dll might change the numbers and then the comment is out-of-sync.

Reply:
I would suggest to turn those that are used into named constants, maybe like

uint64_t kVirtualSize_libtest32dll = 0x1000;

and then use them in the tests.

That's a sure way to find out if numbers in libtest32.dll change and won't be missed when updating.